tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/06_exportCustomers.js

```diff
@@ -0,0 +1,84 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const CustomersPage = require('@pages/BO/customers');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_customers_customers_exportCustomers';
+
+let browser;
+let page;
+let numberOfCustomers = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    customersPage: new CustomersPage(page),
+  };
+};
+
+/*
+Export customers
+Check csv file was downloaded
+Check existence of customers data in csv file
+ */
+describe('Export customers', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    await helper.setDownloadBehavior(page);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO and go to customers page
+  loginCommon.loginBO();
+
+  it('should go to Customers page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToCustomersPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.customersParentLink,
+      this.pageObjects.boBasePage.customersLink,
+    );
+    const pageTitle = await this.pageObjects.customersPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.customersPage.pageTitle);
+  });
+
+  it('should reset all filters and get Number of customers in BO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFirst', baseContext);
+    numberOfCustomers = await this.pageObjects.customersPage.resetAndGetNumberOfLines();
+    await expect(numberOfCustomers).to.be.above(0);
+  });
+
+  it('should export customers to a csv file', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'exportCustomers', baseContext);
+    await this.pageObjects.customersPage.exportDataToCsv();
+    const fileExist = await files.checkFileExistence('customer_', 5000, true, 'csv');
```

Comment:
```suggestion
    const fileExist = await files.isFileExist('customer_', 5000, true, 'csv');
```

Reply:
This change require changing at least 10 tests, and I don't want to do it in this PR

Reply:
:ok_hand: